### PR TITLE
Use native middleware in local runtime

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
# Description

Switch using native middleware for local runtime

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -9,7 +9,18 @@
             "type": "python",
             "justMyCode": false,
             "request": "launch",
-            "name": "VehicleApp",
+            "name": "VehicleApp (native middleware)",
```
Comment (dennismeister93) [inline .vscode/launch.json:12]: just to have it on your mind: this also need to be touched in regards of moving to native:
https://github.com/eclipse-velocitas/vehicle-app-python-sdk/blob/main/.project-creation/templates/.vscode/launch.json

File: .vscode/launch.json [hunk 2]
```diff
@@ -9,7 +9,18 @@
...
+            "program": "${workspaceFolder}/app/src/main.py",
+            "console": "integratedTerminal",
+            "env": {
+                "SDV_MIDDLEWARE_TYPE": "native"
```
Comment (doosuu) [inline .vscode/launch.json:16]: I've never been a fan of these launch configs which set the environment, it couples env vars between here and https://github.com/eclipse-velocitas/devenv-runtimes/pull/64/files#diff-44b589c17495145ec1695f5ff01fde5ea559684154dfb168437ee86e71da3099R36

We could switch this to use an envFile instead which is dynamically written by a preLaunchTask e.g. `velocitas exec runtime-local get-env`

**Review by doosuu — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 2 above)
Comment (MP91) [reply .vscode/launch.json:16]: basically I agree that we have to many places where we define such variables, but we still need to give users the possibility to change it the way they need.

Comment (BjoernAtBosch) [reply .vscode/launch.json:16]: As discussed we will keep this for time being. Will create a follow-up story for fixing this

**Comment by BjoernAtBosch** [comment]
Closing for now - will re-open under other title

**Pull request closed without merging**